src/util/iroha-helpers/util.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Capitalizes string
+ * @param {String} string
+ * @returns {String} capitalized string
+ */
+const capitalize = string => string.charAt(0).toUpperCase() + string.slice(1)
+
+/**
+ * Transfroms hex string to Uint8Array of bytes
+ * @param {String} string string in hex
+ * @returns {Uint8Array}
+ */
+const hexStrToByte = string => {
+  if (!string) {
+    return new Uint8Array()
+  }
+
+  var a = []
+  for (var i = 0, len = string.length; i < len; i += 2) {
+    a.push(parseInt(string.substr(i, 2), 16))
+  }
+
+  return new Uint8Array(a)
+}
+
+/**
+ * Transfroms hex string to Uint8Array of bytes
+ * @param {Uint8Array} uint8arr
+ * @returns {Uint8Array} string in hex
+ */
+const byteToHexString = uint8arr => {
```

Comment:
IMO, it can be replaced with `Buffer.from(uint8arr).toString('hex')` and null guard, and also hexStrToByte can be replaced with `new Uint8Array(Buffer.from(string, 'hex'))` and null guard.

Reply:
I will replace it with Buffers, since @stinger112 updated ED25519 to have polyfills for buffers.